### PR TITLE
Create release notes for 0.30.1

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins doc stage

**Comment by pshipton** [comment]
https://eclipse-openj9.github.io/openj9-docs-staging/895

File: docs/version0.30.1.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!--
+* Copyright (c) 2022, 2022 IBM Corp. and others
```
Comment (keithc-ca) [inline docs/version0.30.1.md:2]: The first year should be 2017 in this repo: see https://github.com/eclipse-openj9/openj9-docs/blob/master/CONTRIBUTING.md.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: docs/version0.30.1.md (hunk 1 above)
Comment (pshipton) [reply docs/version0.30.1.md]: Updated.

**Pull request merged**